### PR TITLE
test intake

### PR DESCRIPTION
**Pull request by noahheller**

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -57,6 +58,7 @@ public class RobotContainer {
       private final Feeder feeder = new Feeder();
       private Climber climber;
       private final IntakeSubsystem intakeSubsystem = new IntakeSubsystem();
+      private final CommandXboxController controller = new CommandXboxController(0);
```
Comment (bengold22) [inline src/main/java/frc/robot/RobotContainer.java]: Move the ID of the Xbox controller to a constant (and probably shouldn't be 0).

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -128,14 +130,15 @@ public void putShuffleboardCommands() {
...
-        SmartShuffleboard.putCommand("Intake", "Start Intake", new StartIntake(intakeSubsystem));
+        SmartShuffleboard.putCommand("Intake", "Start Intake", new StartIntake(intakeSubsystem,5));
         }
     }
 
     private void configureBindings() {
         drivetrain.setDefaultCommand(new Drive(drivetrain, joyleft::getY, joyleft::getX, joyright::getX));
         joyLeftButton1.onTrue(new InstantCommand(() -> drivetrain.setAlignable(Alignable.SPEAKER))).onFalse(new InstantCommand(()-> drivetrain.setAlignable(null)));
         joyRightButton1.onTrue(new InstantCommand(() -> drivetrain.setAlignable(Alignable.AMP))).onFalse(new InstantCommand(()-> drivetrain.setAlignable(null)));
+        intakeSubsystem.setDefaultCommand(new SpinIntake(intakeSubsystem, controller::getLeftY));
```
Comment (bengold22) [inline src/main/java/frc/robot/RobotContainer.java]: I don't expect we'll want this for real but I haven't figured out a good means of defining interim vs final joystick bindings

File: src/main/java/frc/robot/subsystems/IntakeSubsystem.java [hunk 3]
```diff
@@ -1,18 +1,17 @@
...
-import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
-
 public class IntakeSubsystem extends SubsystemBase{
     private final WPI_TalonSRX intakeMotor1;
     private final WPI_TalonSRX intakeMotor2;
 
     public IntakeSubsystem() {
-        this.intakeMotor1 = new WPI_TalonSRX(Constants.INTAKE_MOTOR_1_ID);
-        this.intakeMotor2 = new WPI_TalonSRX(Constants.INTAKE_MOTOR_2_ID);
+        this.intakeMotor1 = new WPI_TalonSRX(11);
```
Comment (bengold22) [inline src/main/java/frc/robot/subsystems/IntakeSubsystem.java]: Revert these hard-coded values

**Review by bengold22 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (noahheller) [reply src/main/java/frc/robot/RobotContainer.java]: done

**Review by bengold22 — CHANGES_REQUESTED** [review]
Remove the default command mapping to spin intake

File: src/main/java/frc/robot/subsystems/IntakeSubsystem.java (hunk 3 above)
Comment (noahheller) [reply src/main/java/frc/robot/subsystems/IntakeSubsystem.java]: done

File: src/main/java/frc/robot/RobotContainer.java (hunk 2 above)
Comment (noahheller) [reply src/main/java/frc/robot/RobotContainer.java]: got rid of defualt command

**Review by bengold22 — APPROVED** [review]

**Pull request merged**